app/src/main/java/helium314/keyboard/latin/settings/SettingsValues.java

```diff
@@ -191,10 +193,14 @@ public SettingsValues(final Context context, final SharedPreferences prefs, fina
         mDoubleSpacePeriodTimeout = res.getInteger(R.integer.config_double_space_period_timeout);
         mHasHardwareKeyboard = Settings.readHasHardwareKeyboard(res.getConfiguration());
         final float displayWidthDp = TypedValueCompat.pxToDp(res.getDisplayMetrics().widthPixels, res.getDisplayMetrics());
-        mIsSplitKeyboardEnabled = prefs.getBoolean(Settings.PREF_ENABLE_SPLIT_KEYBOARD, false) && displayWidthDp > 600; // require display width of 600 dp for split
+        mIsSplitKeyboardPortraitEnabled = prefs.getBoolean(Settings.PREF_ENABLE_SPLIT_KEYBOARD_PORTRAIT, false) && displayWidthDp > 0;
+        mIsSplitKeyboardLandscapeEnabled = prefs.getBoolean(Settings.PREF_ENABLE_SPLIT_KEYBOARD_LANDSCAPE, false) && displayWidthDp > 0;
         // determine spacerWidth from display width and scale setting
-        mSplitKeyboardSpacerRelativeWidth = mIsSplitKeyboardEnabled
-                ? Math.min(Math.max((displayWidthDp - 600) / 600f + 0.15f, 0.15f), 0.35f) * prefs.getFloat(Settings.PREF_SPLIT_SPACER_SCALE, DEFAULT_SIZE_SCALE)
+        mSplitKeyboardPortraitSpacerRelativeWidth = mIsSplitKeyboardPortraitEnabled
+                ? Math.min(Math.max((displayWidthDp) / 0.15f, 0.15f), 0.35f) * prefs.getFloat(Settings.PREF_SPLIT_SPACER_SCALE, DEFAULT_SIZE_SCALE)
```

Comment:
`Math.min(Math.max((displayWidthDp) / 0.15f, 0.15f), 0.35f)` is always `0.35` if your display is more than 1 dp wide.
I think the old method should still be used, so users don't have to re-adjust the split width setting.